docs/langchain.md

```diff
@@ -0,0 +1,94 @@
+# Deploying LangChain Agents on Fixie
+
+[LangChain](https://python.langchain.com/) is a popular framework for developing applications built on top of Large Language Models. Fixie supports LangChain as a first-class citizen. This means you can develop LangChain Agents/Chains/Prompts just as you typically would and then quickly deploy them onto Fixie infrastructure.
+
+## Why Deploy on Fixie?
+
+Deploying on Fixie gives you three major advantages:
+
+1. We host it for you (for free). Simply run `fixie deploy` and your Agent will be hosted in the cloud, complete with a REST API for calling it programatically and a shareable URL that will allow others to test your agent themselves.
+1. No need for an OpenAI key. Fixie will automatically take care of that, and supports 1,000 free requests per day, per user. So if your creation gets popular, you're not stuck footing the bill or asking people to mint their own key.
+1. It allows you to integrate with the rest of the Fixie ecosystem. This means other people in the community will be able to leverage your agent and vice-versa.
+
+## Getting Started
```

Comment:
A user also needs to make a Fixie account, right? And run `fixie auth`?

Reply:
Good catch -- added a note. But now you can run `fixie auth` and if you don't have an account it will create one.